modules/auxiliary/voip/kamailio_dos.rb

```diff
@@ -0,0 +1,47 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  
+  include Msf::Exploit::Remote::Udp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'			 => 'Kamailio SIP Server Off-by-one heap overflow Denial of Service',
+      'Description'	 => %q{
+          A specially crafted REGISTER message with a malformed branch or From tag triggers an off-by-one heap overflow.
+      },
+      'Author'		 => [ 'Alfred Farrugia <alfred[at]enablesecurity.com>' , 'Sandro Gaucci <sandro[at]enablesecurity.com>' , 'Carlos Perez <cp.ardanaz[at]gmail.com>' , 'Jon Uriona <jon.uriona[at]gmail.com>' ],
```

Comment:
Please linewrap array elements.

Also, traditionally a comment is added after each person's name, with a description of what they contributed.

```ruby
'Author' => [
  'Alfred Farrugia <alfred[at]enablesecurity.com>' , # Discovery and PoC
  'Sandro Gaucci <sandro[at]enablesecurity.com>' , # Discovery and PoC
  'Carlos Perez <cp.ardanaz[at]gmail.com>', # MVP
  'Jon Uriona <jon.uriona[at]gmail.com>' # Metasploit
],
```